src/borg/testsuite/archiver.py

```diff
@@ -1298,9 +1298,12 @@ def test_list_size(self):
         self.cmd('init', self.repository_location)
         test_archive = self.repository_location + '::test'
         self.cmd('create', '-C', 'lz4', test_archive, 'input')
-        output = self.cmd('list', '--format', '{size} {csize} {path}{NL}', test_archive)
-        size, csize, path = output.split("\n")[1].split(" ")
+        output = self.cmd('list', '--format', '{size} {csize} {dsize} {dcsize} {path}{NL}', test_archive)
+        size, csize, dsize, dcsize, path = output.split("\n")[1].split(" ")
         assert int(csize) < int(size)
+        assert int(dcsize) < int(dsize)
+        assert int(dsize) <= int(size)
+        assert int(dcsize) <= int(csize)
```

Comment:
So, trying to understand the issue for archives, would be something similar to this (thinking on the worst case scenario):
Have a big file (say 1GB video1.mp4), and for some reason I decide to copy it as video2.mp4 and then create an arvhice (note that the video had never been uploaded before). now, the entire 1GB is completely deduplicated (all the chunks have refcount>1) and thus, it would not be taken into account for the calculation of deduplicated size?